portal/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -0,0 +1,101 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.ContentTypes.`application/json`
+import akka.http.scaladsl.model.{ HttpEntity, StatusCodes }
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import cromwell.pipeline.datastorage.dto.user.DeactivateUserRequestByEmail
+import cromwell.pipeline.datastorage.dto.{ UserDeactivationByEmailResponse, UserDeactivationByIdResponse, UserId }
+import cromwell.pipeline.service.UserService
+import cromwell.pipeline.utils.auth.SecurityDirective
+import cromwell.pipeline.{ AuthConfig, ExpirationTimeInSeconds }
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{ Matchers, WordSpec }
+import pdi.jwt.JwtAlgorithm
+import play.api.libs.json.Json
+
+import scala.concurrent.Future
+
+class UserControllerTest extends WordSpec with Matchers with MockFactory with ScalatestRouteTest {
+
+  private val UserService = stub[UserService]
```

Comment:
`UserService` -- First capital letter is redundant.

Reply:
done)